src/modules/webpack.ts

```diff
@@ -0,0 +1,74 @@
+import { CommandStringBuilder } from "../utilities/command-string-builder";
+import { OptionStringBuilder } from "../utilities/option-string-builder";
+import { Dir } from "./dir";
+import { FrontendPath } from "./frontend-path";
+import shell from "shelljs";
+import { NodeClean } from "./node-clean";
+import { WebpackPublish } from "./webpack-publish";
+import { NodeRestore } from "./node-restore";
+import { Echo } from "./echo";
+import child_process from "child_process";
+
+// -----------------------------------------------------------------------------------------
+// #region Constants
+// -----------------------------------------------------------------------------------------
+
+const COMMAND = new CommandStringBuilder("npm", "run", "start");
+
+// #endregion Constants
+
+// -----------------------------------------------------------------------------------------
+// #region Public Functions
+// -----------------------------------------------------------------------------------------
+
+const Webpack = {
+    cmd(): CommandStringBuilder {
+        return COMMAND;
+    },
+    description(): string {
+        return `Runs the webpack project (via ${this.cmd()}}) found in ${FrontendPath.projectDir()}`;
+    },
+    run(clean: boolean = false, restore: boolean = false) {
+        Dir.pushd(FrontendPath.projectDir());
+
+        // Webpack won't be able to start up without restoring after a clean
```

Comment:
Thanks for the comment!  Does it make sense to move this down to line 41-43 where the variable is being used?  Don't have the domain knowledge around it exactly, but I would think the fact that 42 bails that the comment would be related to it.

Reply:
I could see it either way 😛 Putting it above the variable gives more context to why the variable is being defined/set, but maybe the actual 'result' of its use is where it could be of more use. I'll move it 👍